docs/book/src/multiversion-tutorial/testdata/project/internal/webhook/v1/cronjob_conversion.go

```diff
@@ -22,5 +22,18 @@ method called `Hub()` to serve as a
 We could also just put this inline in our `cronjob_types.go` file.
 */
 
+import (
+	batchv1 "tutorial.kubebuilder.io/project/api/v1" // Import CronJob from API package
+)
+
+/*
+ConversionCronJob wraps the API's CronJob and implements the Hub interface for conversion.
+*/
+
+// ConversionCronJob wraps the existing CronJob type from the API.
+type ConversionCronJob struct {
```

Comment:
I don't think this works. To the best of my knowledge Hub() still has to be implemented on the API types today

Reply:
I will put that back in the api dir.
thank you

Reply:
Done, good catcher!